.github/renovate.json

```diff
@@ -13,5 +13,14 @@
       "datasourceTemplate": "github-tags",
       "extractVersionTemplate": "^v(?<version>.*)$"
     }
+  ],
+  "github-actions": {
+    "fileMatch": ["action.yml"]
+  },
+  "packageRules": [
+    {
+      "matchManagers": ["github-actions"],
+      "matchFiles": ["action.yml"]
+    }
```

Comment:
This is not required, renovate already knows that an action.yml in the root of a repo is a github composite action.

Reply:
Yeah, this is something I had to ask you. Thanks for letting me know.